aiida/restapi/translator/calculation/__init__.py

```diff
@@ -91,10 +92,11 @@ def __init__(self, **kwargs):
         }
 
     @staticmethod
-    def get_files_list(dirobj, files=None, prefix=None):
+    def get_files_list(nobj, dirobj=None, files=None, prefix=None):
         """
         Get list of all files present in given directory.
```

Comment:
Can you explain a bit here the purpose of the function, i.e. should one provide a node object *or* a directory object?

Reply:
I modified the docstring to:

        Return the list of all files contained in the node object repository
        If a directory object `dir_obj` of the repository is passed, get the list of all files
        recursively in the specified directory

Reply:
thanks!